From eba578cf9a666763bf297a387b781b669893b69d Mon Sep 17 00:00:00 2001
From: lyctw <lyctw.ee@gmail.com>
Date: Sun, 15 May 2022 21:05:49 +0800
Subject: [PATCH] enable debug info

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 062883e..704d0a6 100644
--- a/Makefile
+++ b/Makefile
@@ -277,7 +277,7 @@ GENFLAGS	+=	$(libsbiutils-genflags-y)
 GENFLAGS	+=	$(platform-genflags-y)
 GENFLAGS	+=	$(firmware-genflags-y)
 
-CFLAGS		=	-g -Wall -Werror -ffreestanding -nostdlib -fno-stack-protector -fno-strict-aliasing -O2
+CFLAGS		=	-ggdb3 -Wall -Werror -ffreestanding -nostdlib -fno-stack-protector -fno-strict-aliasing -O0
 CFLAGS		+=	-fno-omit-frame-pointer -fno-optimize-sibling-calls -mstrict-align
 # enable -m(no-)save-restore option by CC_SUPPORT_SAVE_RESTORE
 ifeq ($(CC_SUPPORT_SAVE_RESTORE),y)
-- 
2.35.1

