From 193250ff645997d9c4b4f7cdaf6f3a9f1bd200a5 Mon Sep 17 00:00:00 2001
From: Yu Chien Peter Lin <peterlin@andestech.com>
Date: Mon, 10 Jun 2024 14:42:44 +0800
Subject: [PATCH 1/8] spl: Kconfig: Add SPL_OPTEE_IMAGE_LOAD_ADDR

---
 common/spl/Kconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/common/spl/Kconfig b/common/spl/Kconfig
index 6405374bcc..2c87a8e67a 100644
--- a/common/spl/Kconfig
+++ b/common/spl/Kconfig
@@ -1465,6 +1465,11 @@ config SPL_OPTEE_IMAGE
 	  OP-TEE is an open source Trusted OS which is loaded by SPL.
 	  More detail at: https://github.com/OP-TEE/optee_os
 
+config SPL_OPTEE_IMAGE_LOAD_ADDR
+	hex "OP-TEE Trusted OS image load address"
+	help
+	  Load address of the OP-TEE binary.
+
 config SPL_OPENSBI
 	bool "Support RISC-V OpenSBI"
 	depends on RISCV && SPL_RISCV_MMODE && RISCV_SMODE
-- 
2.34.1

